include/file.iol

```diff
@@ -77,6 +77,18 @@ type ListResponse:void {
 interface FileInterface {
 RequestResponse:
 	/**!
+	 * Constructs an absolute path to the target file or directory. 
+	 * Can be used to construct an absolute path for new files that does not exist yet.
+	 */
+	getAbsolutePath( string )( string ),
```

Comment:
I'd probably call it toAbsolutePath, following Java in toString, toAbsolutePath, etc.

@jolie/developers Comments on the convention for transformer operations? I don't have *very* strong feelings here. (yet?) :-)